tests/python_functional/functional_tests/source_drivers/network_source/proxyprotocol/test_pp_with_reconnect.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+#############################################################################
+# Copyright (c) 2020 One Identity
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 as published
+# by the Free Software Foundation, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+#
+# As an additional exemption you are allowed to compile & link against the
+# OpenSSL libraries as published by the OpenSSL project. See the file
+# COPYING for details.
+#
+#############################################################################
+import time
+
+NUMBER_OF_MESSAGES = 5
+
+
+def test_pp_with_reconnenct(config, port_allocator, syslog_ng, loggen):
+    network_source = config.create_network_source(ip="localhost", port=port_allocator())
+    file_destination = config.create_file_destination(file_name="output.log")
+    config.create_logpath(statements=[network_source, file_destination])
+
+    syslog_ng.start(config)
+
+    loggen.start(
+        network_source.options["ip"], network_source.options["port"], inet=True,
+        rate=1, active_connections=1, reconnect=True, interval=30, number=NUMBER_OF_MESSAGES,
+    )
+
+    syslog_ng.stop()
+    time.sleep(1)
```

Comment:
Please remove this time.sleep(1) and time.sleep(5) expressions from the testcase.
Testcase will pass without them also. Unfortunately It will highly slow down the full functional testcase run.

Reply:
Thanks for the fix.